Improved Readability + en_gb version

edent opened the pull request:
* Fixes issues with https://github.com/ContributorCovenant/contributor_covenant/pull/505
* Fixes #498

edent committed:
Improved Readability + en_gb version

* Fixes issues with https://github.com/ContributorCovenant/contributor_covenant/pull/505
* Fixes #498

bahmady commented:
@ianbollinger pointed out on https://github.com/ContributorCovenant/contributor_covenant/pull/510 that "education" and "socioeconomic status" aren't in this suggested edit. 

I've just found that "education" and "socioeconomic status" were only added on November 23rd: https://github.com/ContributorCovenant/contributor_covenant/commit/9815dbb8b9acbd02f2e3932f89b7a9bfc769643c

^ the version I pulled to edit from alphagov/open-standards (https://github.com/alphagov/open-standards/pull/52) didn't have these additions. 

From a style point of view I can't see any reason at all not to add them back.

edent committed:
Update code-of-conduct.gb.md

As per https://github.com/ContributorCovenant/contributor_covenant/pull/505#issuecomment-362923162

edent committed:
Update code-of-conduct.md

As per https://github.com/ContributorCovenant/contributor_covenant/pull/505#issuecomment-362923162

edent commented:
@bahmady @ianbollinger - thanks, I've added those back in.

bahmady commented:
Hello—

let me know if there's anything else I can help with on this one 😃
